### ISSUE TITLE
/usr/local/bin/docker-entrypoint.sh: line 3: [[-f: command not found

### ISSUE THREAD

**Issue by crazy-max** [renovatebot/docker-renovate#17]
Hi,

Coming back from #15, when I run the following docker command, renvoate can't read the `config.js` file:

```
$ docker run --rm -i --name renovate -e TZ=Europe/Paris -e GITHUB_TOKEN=**** -e GITHUB_COM_TOKEN=**** -v /home/renovate/config.js:/usr/src/app/config.js renovate/renovate:19 --token=**** --log-level=info --dry-run=false --print-config=false --base-dir=/tmp/renovate --username=**** --password=****
/usr/local/bin/docker-entrypoint.sh: line 3: [[-f: command not found
```

Was working with `org.opencontainers.image.version=19.219.16` but failed with `org.opencontainers.image.version=19.221.0-slim`.

**Comment by viceice** [comment]
@crazy-max Do you have a idea what's wrong with the entrypoint?

**Comment by viceice** [comment]
should be fixed by 931dac002640a6cb3d2687750598cb03e84c4b3b

@crazy-max Can you verify the latest slim image after the master build pushed new docker tag?

**Comment by crazy-max** [comment]
@viceice Yep LGTM
